lib/derive/src/parse.rs

```diff
@@ -13,6 +13,11 @@ pub enum WasmerAttr {
         identifier: Option<LitStr>,
         span: Span,
     },
+    #[ cfg(feature="async") ]
+    Yielder {
+        identifier: Option<LitStr>,
+        span: Span,
+    }
```

Comment:
Minor nit (I haven't review all the derive macro changes yet) but we probably want to unconditionally parse this attribute and error if the feature isn't enabled or otherwise generate code that will provide a useful error message in that case. Don't worry about doing all that work now for this experimental PR, just writing this down for future reference.